.gitignore

```diff
@@ -12,6 +12,12 @@ node_modules
 # production
 dist
 
+# deployment
+.travis/58231b16fdee45a03a4ee3cf94a9f2c3
```

Comment:
For future reference this file comes from https://github.com/RedHatInsights/insights-frontend-builder-common/blob/master/src/bootstrap.sh#L8